packages/hash/frontend/src/pages/[shortname]/types/entity-type/[entity-type-id].page/edit-bar-reusable.tsx

```diff
@@ -0,0 +1,112 @@
+import { Box, Collapse, Container, Stack, Typography } from "@mui/material";
+import { ReactNode } from "react";
+
+import { PencilSimpleLine } from "../../../../../shared/icons/svg";
+import { Button, ButtonProps } from "../../../../../shared/ui/button";
+import { useFreezeScrollWhileTransitioning } from "./edit-bar";
+
+const EditBarContents = ({
```

Comment:
This is almost identical to the one used by type editor – i'd like to avoid introducing this duplication here, even temporarily. Could you add a `buttonsDisabled` prop to avoid the dependency on hook form as in the other version?

Reply:
Instead of adding `buttonsDisabled`, I used button props to pass loading-disabled statuses to the buttons.

Addressed here https://github.com/hashintel/hash/pull/1734/commits/428ad17249846b12e4a572423412871017faafef